doc/INSTALL

```diff
@@ -246,353 +217,375 @@ General notes for the installation of the 4.2 release of KVIrc
     to "/usr/lib/libgsm.a", you might create the shared library manually
     by running:
 
-    # cd /usr/lib
-    # ld --whole-archive -shared -o libgsm.so.1 libgsm.a
-    # ln -s libgsm.so.1 libgsm.so
-    # ldconfig
-
-    If you don't have it installed at all, you might want to have a look
-    on your distribution CD, or download it from the web.
-
-
-    - (Optional) If you want the /snd plugin to play various audio formats,
-      you either need a running artsd (nowadays replaced by the phonon
-      framework), a running ESounD daemon, or a reasonably recent
-      audiofile library. Without these KVIrc will be only able to play *.au
-      files.
-
-    - You need the dynamic linker interface library libdl.so.
-
-    This is usually installed on your system, so don't care until configure
-    complains about it. Some system have the interface builtin in libc. The
-    configure script can detect this.
+    $ cd /usr/lib
+    $ ld --whole-archive -shared -o libgsm.so.1 libgsm.a
+    $ ln -s libgsm.so.1 libgsm.so
+    $ ldconfig
+
+  ## Phonon framework - (Preferred)
+    
+  If you want the /snd plugin to play various audio formats,
+  you either need the phonon framework, a running ESounD daemon,
+  or a reasonably recent audiofile library.
+  Without these KVIrc will be only able to play *.au files.
+
+  - You will need the dynamic linker interface library libdl.so.
+
+  This is usually installed on your system, so don't care until configure
+  complains about it. Some system have the interface builtin in libc. The
+  configure script can detect this.
+    
+-------------------------------------------------------------------------------
+  4. Getting the source code:
+-------------------------------------------------------------------------------
+
+  The KVIrc development code is located at https://github.com/kvirc/KVIrc
+  There are several ways to get KVIrc's source code, if you compile often,
+  Preferable way is to clone the repository, this allows for the most flexibility.
+  Alternatively if you prefer. you can also download a tarball.
+    
+    $ mkdir /home/setup && cd /home/setup
+0.  $ git clone https://github.com/kvirc/KVIrc.git
+
+    Keeping the git clone up-to-date  is simple.
+    
+    $ git pull --rebase upstream master
+
+    Or download the latest release tarball
+
+    $ mkdir /home/setup && cd /home/setup
+1.  $ wget https://github.com/kvirc/KVIrc/archive/master.tar.gz -O kvirc.git.tar.gz
+
+-------------------------------------------------------------------------------
+  5. Configuring KVIrc
+-------------------------------------------------------------------------------
+
+  Since KVIrc 4.2 uses CMake as build system, we have to use it to create
+  the Makefile's rules for make program.
+  To do it, we encourage the "out-of-source" building: build all files
+  without dirtying the sources directory.
+
+    $ cd /home/setup
+    $ mkdir release
+    $ cd release
+    $ cmake [your options] .. 
```

Comment:
That info has always been like that mentioned like that exactly.

And as I understand it you need both since they cover very specific scenarios one that export blablablah wont cover.

If you mean condensing the usage down to one section so that its not repeated Ill have another look.

export path wont work for telling cmake to use qt 4.8.5 when you also have 4.8

. In any case I dont really care, if you think your proposal will cover and work the way its intended then